src/qoi/include/grins/rayfire_mesh.h

```diff
@@ -39,24 +39,44 @@ namespace GRINS
     
   //! RayfireMesh
   /*!
-  This class creates a 1D mesh of EDGE2 elements to allow
-  for integration of functions along a line across the domain
+  This class performs a rayfire across a given mesh.
+  The user supplies an origin point on the mesh boundary,
+  and spherical angle(s) theta (and phi for 3D) to describe the direction of the rayfire.
+  As is standard in spherical coordinates, theta is the angle from the x-axis in the xy-plane,
+  with counterclockwise being positive. -2&pi; &le; theta &le; +2&pi;.
+  Phi is the angle from the xy-plane, with phi>0 being in the positive half of the z-plane.
+  -2&pi; &le; phi &le; +2&pi;
```

Comment:
phi goes from -\pi to \pi, right?

Also, OK for now, but for future info, you can enclose LaTeX statements in Doxygen comments with `\f$ <latex> \f$` or, for a full equation, `\f[ <latex> \f]`.